packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -37,8 +37,6 @@ export interface DrawerPanelContentProps extends React.HTMLProps<HTMLDivElement>
   };
   /** Color variant of the background of the drawer panel */
   colorVariant?: DrawerColorVariant | 'light-200' | 'default';
-  /** @hide Internal ref to drawer content */
-  drawerContentRef?: React.RefObject<HTMLDivElement>;
```

Comment:
One question I have, can we remove props that have been hidden without it being a breaking change?

Reply:
I think that _should_ be okay - but I will defer to @tlabaj here.

Reply:
the hidden props are internal, so consumer should no be using them.  I think it is ok to remove.